sdk-core/custom_tokens.md

```diff
@@ -0,0 +1,38 @@
+## CustomToken
+
+```
+dto CustomToken:
+    // E.g. "FOO-abcdef", "EGLD".
+    identifier: string;
+    nonce: number;
+```
+
+## TokenComputer
+
+```
+class CustomTokenComputer:
+    // The constructor is not captured by the specs; it's up to the implementing library to define it.
+
+    // Returns token.nonce == 0.
+    is_fungible_token(token: Token): boolean;
```

Comment:
do all methods described here can error? for example, what happens if I call the `extract_token_identifier_from_extended_identifier` with the `abcd` string?

Reply:
Very good point! Will specify this in a separate PR :+1:

Reply:
Noted here: https://github.com/multiversx/mx-sdk-specs/issues/4.